### DISCUSSION TITLE
Disable Prettier?

### DISCUSSION THREAD

**Discussion by PStenbumling** [hassio-addons/addon-vscode#585] [General]
Hi!

Can i disable Prettier on save ? 

have a 5k lines lovelace.yaml 
Is this the one to untick?
![image](https://user-images.githubusercontent.com/39261692/224962511-1fa68a02-8f12-4b63-b93a-cb11ed079cdc.png)
![image](https://user-images.githubusercontent.com/39261692/224961844-df2ec1a1-1287-4967-a328-606ed26c0790.png)

![image](https://user-images.githubusercontent.com/39261692/224962060-b5cab906-1be9-45e0-b287-c66e120c252f.png)

**Comment by frenck** [comment]
This is more a question for Prettier, as it is not related to the add-on.

You can also select a formatter or disable formatting al together.
